script/Ciel/Day 10/10_01_CIEL10_5B.txt

```diff
@@ -50,6 +50,7 @@ It'd be dangerous to leave you alone anyway,
 [sha:d150647417e23a9125a90ee3455ed159e6771659]{
 -- Page 358, Offset 28038.
 -- これでも先輩ですから。やるとなったら吸血鬼探しのイロハをたっぷり教えこんであげましょう！」
+// apparently get stuck in didn't made sense in context
```

Comment:
Submitter comment
>I am entirely failing to parse what 'once we get stuck in' is supposed to mean. Once we get stuck in [to the state of being-in-the-process-of-vampire-hunting]? Or something? That's my best attempt at a reconstruction of the intended meaning, but it's a low-confidence one; mostly I'm just confused.

Reply:
Hmm, seems fairly clear to me? Maybe just not familiar with the expression?

Reply:
I'm gonna be honest, I don't think that "get stuck in" is much of an Americanism. Reads to me like a British expression. "Once we get into it" might be more American.

Reply:
https://dictionary.cambridge.org/dictionary/english/get-stuck-in
Seems to be a Britishism
As the Britishism-eliminator, I'll apologize for missing it.

Reply:
going with Ben's suggestion then